packages/libp2p/src/libp2p.ts

```diff
@@ -49,9 +47,11 @@ export class Libp2pNode<T extends ServiceMap = Record<string, unknown>> extends
   public keychain: KeyChain
   public metrics?: Metrics
   public services: T
+  public logger: ComponentLogger
 
   public components: Components
   #started: boolean
+  readonly #log: Logger
```

Comment:
Any particular rationale that this is a private field as opposed to the components? or should they be private as well?

Reply:
Anything not in the libp2p interface should be private.

Reply:
Component log fields are private now.